content/docs/user-guide/assets/scene-settings/_index.md

```diff
@@ -1,33 +1,41 @@
-* When the scene file is processed, there is a point where Python scripts can be called to modify the scene settings loaded in memory. This is highest priority because this same logic might read the current settings and choose which to preserve and which settings to override.
-
-The source Scene Settings tracked in the `.assetinfo` file is best edited with the [Scene Settings tool](/docs/user-guide/assets/scene-settings/interface) within the O3DE Editor.
+## Source and product assets
 
-## Scene Settings topics
+*Scene source assets* (meshes, actors, motions, and PhysX colliders) that are created in digital content creation (DCC) applications must be processed as runtime optimized *product assets* to be used in your project. To provide the broadest support for DCC applications and scene file formats, O3DE integrates the [Open Asset Import Library](https://github.com/assimp/assimp). O3DE uses `.fbx` as its primary format for scene source assets. O3DE also supports `.stl`, and support for `.gltf` is enabled and currently in development.
 
-The topics in this section provide general information about the Scene Settings interface, product asset output types and modifiers that you can use to customize how 3D scene source assets are processed.
+## Scene settings
+
+When source assets are placed in a scan directory, [Asset Processor](../../assets/asset-processor) detects the new or modified files, determines the contents of the files, and then processes the data using *scene settings* that tell Asset Processor how to process the files.
+
+Scene settings can come from several places:
+
+* Default scene settings logic exists for all supported asset types. Asset Pipeline uses this default logic to process scene files when no other settings exist.
+* Soft naming conventions on the mesh and skeleton nodes in the source asset can be used to override the default logic. For example, a node with the suffix `_lod1` is treated as a level of detail (LOD).
+* User generated `.assetinfo` files override the default logic, and can override soft naming conventions. These sidecar files have the same name as their relative source asset file, but have a `.assetinfo` extension. For example, `myCharacter.fbx.assetinfo` contains the the scene settings for `myCharacter.fbx`. User generated `.assetinfo` files are created with the [Scene Settings](scene-settings) tool.
+* Some jobs, usually from a [Python Asset Builder](/docs/user-guide/assets/builder/), might output procedurally generated `.assetinfo` files. These files take priority over user generated `.assetinfo` files because the procedural system can read the source file when generating scene settings and choose to override them.

```

Comment:
Suggestion to avoid deep-linking to a topic here.

```suggestion
* Some jobs output procedurally generated `.assetinfo` files as part of asset processing. These files take priority over user generated `.assetinfo` files because the procedural system can read the source file when generating scene settings and choose to override them.

```